### PR TITLE
Mixed Models - fixes

### LINKED ISSUE #957: bayes failure for linear mixed
* JASP version:

* OS name and version:0:13.1
* Analysis: bayes linear mixed
* Bug description:
won't run has message
This analysis terminated unexpectedly.

Error in dimnamesGets(x, value): invalid dimnames given for “dgCMatrix” object

Stack trace

To receive assistance with this problem, please report the message above at: https://jasp-stats.org/bug-reports
* Expected behaviour:

#### Steps to reproduce:
1. Go to analysis tab
2. Click on '...bayes linear mixed.'
dependent proptn
fixed wiithin
random pno'
This analysis terminated unexpectedly.
Error in dimnamesGets(x, value): invalid dimnames given for “dgCMatrix” object
Stack trace

To receive assistance with this problem, please report the message above at: https://jasp-stats.org/bug-reports
* Expected behaviour:

### PR DESCRIPTION
**Pull request by FBartos**
fixes: https://github.com/jasp-stats/INTERNAL-jasp/issues/989 (after updating `afex `package to the current CRAN version)
fixes: https://github.com/jasp-stats/jasp-issues/issues/957 (after updating `stanova` package to the current GitHub version)

### CODE REVIEW

File: JASP-Engine/JASP/R/MixedModelsCommon.R [hunk 1]
```diff
@@ -2138,7 +2143,8 @@
         warmup            = options$warmup,
         adapt_delta       = options$adapt_delta,
         control           = list(max_treedepth = options$max_treedepth),
-        seed              = if (isTRUE(options[["setSeed"]])) options[["seed"]]
+        seed              = if (isTRUE(options[["setSeed"]])) options[["seed"]],
```
Comment (vandenman) [inline JASP-Engine/JASP/R/MixedModelsCommon.R]: shouldn't we maybe just add a function in common like:
```r
.getSeedJASP <- function(options) {
  if (is.list(options) && c("setSeed", "seed") %in% names(options)) {
    if (isTRUE(options[["setSeed"]]))
      return(options[["seed"]])
  }
```
and then call this one all over? That way the seed calling is all located in one place.

**Comment by vandenman** [comment]
The code looks fine and jasp-stats/INTERNAL-jasp#989 seems fixed. 
I have no idea how this PR fixes jasp-stats/jasp-issues#957 though, although that might be because that issue is rather vague... @FBartos any idea what I should check to verify that the issue is fixed?

**Comment by FBartos** [comment]
the issue is fixed by updating the `stanova` package. There seemed to be some mishandling of the aggregated binomial models withing it. You can verify it by fitting an aggregated binomial BGLMM.

File: JASP-Engine/JASP/R/MixedModelsCommon.R (hunk 1 above)
Comment (FBartos) [reply JASP-Engine/JASP/R/MixedModelsCommon.R]: added

**Comment by kornbrot** [comment]
greart. so has stanova package been fixed and updated?
is there an update link? or should I re-install?

**Comment by FBartos** [comment]
Hi @kornbrot , please do not do anything yet. This is being fixed only in the internal version. I will let you know when the fixed version is out and publically accessible.

**Review by vandenman — APPROVED** [review]

**Comment by kornbrot** [comment]
Many thanks

Sent from my iPhone

On 18 Sep 2020, at 13:34, FBartos <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi @kornbrot<https://github.com/kornbrot> , please do not do anything yet. This is being fixed only in the internal version. I will let you know when the fixed version is out and publically accessible.

File: JASP-Engine/JASP/R/common.R [hunk 2]
```diff
@@ -3074,3 +3074,9 @@ postProcessModuleInstall <- function(moduleLibraryPath)
   #             "The argument options should be the options list from QML.",
   #             "Ensure that the SetSeed{} QML component is present in the QML file for this analysis."))
 }
+
+.getSeedJASP <- function(options) {
```
Comment (TimKDJ) [inline JASP-Engine/JASP/R/common.R:3078]: Unit tests found a syntax error -- one `{` is not closed

**Review by TimKDJ — CHANGES_REQUESTED** [review]

File: JASP-Engine/JASP/R/common.R (hunk 2 above)
Comment (FBartos) [reply JASP-Engine/JASP/R/common.R:3078]: ok, thanks. Fixed

**Comment by boutinb** [comment]
@TimKDJ Can we merge this PR now?

**Pull request merged**